index.html

```diff
@@ -9298,6 +9298,279 @@
         </section>
       </div>
     </section>
+    <section id="rtcquicstream-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+        <pre class="idl">
+        [ Exposed=Window ]
+interface RTCQuicStream {
+    readonly        attribute RTCQuicTransport    transport;
+    readonly        attribute RTCQuicStreamState  state;
+    readonly        attribute unsigned long       bufferedAmount;
+    void                  close ();
+    void                  write (ArrayBuffer data);
```

Comment:
For this version of the PR, I recommend write() being "infinite buffer" and deal with better buffering control in a later PR.

Reply:
Removed text implying read buffering.